cpp/log/log_verifier.h

```diff
@@ -44,6 +44,8 @@ class LogVerifier {
         return "Inconsistent timestamps.";
       case INVALID_SIGNATURE:
         return "Invalid signature.";
+      case INVALID_MERKLE_PATH:
+        return "Invalid Merkle path.";
```

Comment:
And that's why I don't like the "default:" (removing it would give a compile error, with the compilers we're using).